benchmarks/free_surface_tractions/viscoelastic/README

```diff
@@ -54,3 +54,12 @@ The solutions match well in 3-D. In 2-D, the geometries of the loading
  function are different (Cartesian in ASPECT vs cylindrical analytically).
  As such, the agreement in 2-D breaks down for small r0 (load width) or
  if the right boundary is placed further away.
+ 
+The fit between the numerical and the analytical solution can be improved
+ by choosing short elastic (and therewith even shorter numerical) time
+ steps. The shorter the elastic time step is, the more instantaneous is 
+ the surface response to loading and unloading. When using a very small 
+ time step in the analytical solution it can be seen that the response 
+ is indeed instantaneous, favoring small elastic and computational time
+ steps for an accurat numerical solution as well. 
```

Comment:
```suggestion
 steps for an accurate numerical solution as well. 
```

Reply:
I was going to merge this and fix it up myself, rather than sending @EstherHeck through another round of back-and-forth...